widgets/control-the-widget-state/src/runtime/widget.tsx

```diff
@@ -0,0 +1,169 @@
+/**
+  Licensing
+
+  Copyright 2021 Esri
+
+  Licensed under the Apache License, Version 2.0 (the "License"); You
+  may not use this file except in compliance with the License. You may
+  obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+  implied. See the License for the specific language governing
+  permissions and limitations under the License.
+
+  A copy of the license is available in the repository's
+  LICENSE file.
+*/
+// import { React, IMDataSourceInfo, DataSource, DataSourceManager, DataSourceStatus, FeatureLayerQueryParams, AllWidgetProps, DataSourceComponent } from 'jimu-core';
+const { useState, useEffect, useRef } = React;
```

Comment:
Should this line be placed under the import React

Reply:
Thanks Mikey. The commented line has been removed as it was unnecessary.